docker/Dockerfile

```diff
@@ -14,12 +14,17 @@ RUN go get -v github.com/readium/readium-lcp-server/lcpencrypt
 
 FROM phusion/baseimage:focal-1.1.0 As base
 
+COPY ./docker/simplified_app.sh /ls_build/
+COPY --chown=1000:1000 poetry.lock pyproject.toml /var/www/circulation/
```

Comment:
Then after this line you can run your new script that installs the dependencies using poetry